### PR TITLE
Issue #215: Fix PHP 7.x compatibility.

### LINKED ISSUE #215: Fix PHP 7.x compatibility
This PR fixes PHP 7.x compatibility and at the same time issues reported in https://www.drupal.org/node/2999011 and https://www.drupal.org/node/3012715

### PR DESCRIPTION
**Pull request by drupol**
Fix #215

### CODE REVIEW

File: atomium/templates/table/table.component.inc [hunk 1]
```diff
@@ -79,89 +81,141 @@ function atomium_preprocess_table(array &$variables, $hook) {
...
 
   // Format the table footer rows.
-  $variables['rows'] = _atomium_table_format_rows('body', $variables['rows'], $header, $ts, FALSE, $variables);
+  $variables['rows'] = _atomium_table_format_rows(
+    'body',
+    $variables['rows'],
+    $header,
+    $ts,
+    FALSE,
+    $variables
```
Comment (aritomelo) [inline atomium/templates/table/table.component.inc:113]: As we follow Drupal CS, here is missing a comma. Check [here](https://www.drupal.org/docs/develop/standards/coding-standards#array).

File: atomium/templates/table/table.component.inc [hunk 2]
```diff
@@ -79,89 +81,141 @@ function atomium_preprocess_table(array &$variables, $hook) {
...
 
   // Format the table footer rows.
-  $variables['footer'] = _atomium_table_format_rows('footer', $variables['footer'], $header, $ts, FALSE, $variables);
+  $variables['footer'] = _atomium_table_format_rows(
+    'footer',
+    $variables['footer'],
+    $header,
+    $ts,
+    FALSE,
+    $variables
```
Comment (aritomelo) [inline atomium/templates/table/table.component.inc:123]: ... Comma missing.

**Review by aritomelo — DISMISSED** [review]
Check my comments, the rest looks great.

File: atomium/templates/table/table.component.inc (hunk 1 above)
Comment (drupol) [reply atomium/templates/table/table.component.inc:113]: It's a function call, adding a comma there will trigger a syntax error :-)

File: atomium/templates/table/table.component.inc (hunk 2 above)
Comment (drupol) [reply atomium/templates/table/table.component.inc:123]: Same here.

**Review by aritomelo — APPROVED** [review]
Ready to be merged.

**Pull request merged**